tests/unit/mock/config/parser/base/paloalto_panos/panos_full_received.py

```diff
@@ -0,0 +1,3202 @@
+from netutils.config.parser import ConfigLine
+
+data = [
+    ConfigLine(config_line="config {", parents=()),
+    ConfigLine(config_line="  mgt-config {", parents=("config {",)),
+    ConfigLine(config_line="    users {", parents=("config {", "  mgt-config {")),
+    ConfigLine(config_line="      admin {", parents=("config {", "  mgt-config {", "    users {")),
+    ConfigLine(config_line="        phash *;", parents=("config {", "  mgt-config {", "    users {", "      admin {")),
+    ConfigLine(
+        config_line="        permissions {", parents=("config {", "  mgt-config {", "    users {", "      admin {")
+    ),
+    ConfigLine(
+        config_line="          role-based {",
+        parents=("config {", "  mgt-config {", "    users {", "      admin {", "        permissions {"),
+    ),
+    ConfigLine(
+        config_line="            superuser yes;",
+        parents=(
+            "config {",
```

Comment:
hmm, shouldn't the line just be 
```
            "config {",
```

and in below lines the parent be the same as above?

Reply:
The unittest passes, so unless that's wrong I don't think so? Unless I'm misunderstanding something? based on the other brace configs. I saw the entire hierarchy was included for each line.

Reply:
> The unittest passes, so unless that's wrong I don't think so?

How were the unittests created? Did you generate it by hand? If it was ran by code, the unittest only tests that it is what it was originally. 

> based on the other brace configs. I saw the entire hierarchy was included for each line.

Which brace config are you referring to? I only see F5 as brace, which is in the format I was expecting. https://github.com/networktocode/netutils/blob/develop/tests/unit/mock/config/parser/base/bigip_f5/bigip_full_received.py#L726

Reply:
Moving to conversion to set format.